plugin.py

```diff
@@ -1,11 +1,16 @@
+from LSP.plugin.core.typing import Any, Dict, Literal, Set, Union
+from LSP.plugin.core.url import uri_to_filename
+from lsp_utils import NpmClientHandler
 import os
 import posixpath
 import re
 import sublime
 import webbrowser
 
-from LSP.plugin.core.url import uri_to_filename
-from lsp_utils import NpmClientHandler
+EslintValidateOn = Literal['on']
+EslintValidateOff = Literal['off']
+EslintValidateProbe = Literal['probe']
+EslintValidate = Union[EslintValidateOn, EslintValidateOff, EslintValidateProbe]
```

Comment:
From the mypy docs about Literal:

> Literals containing two or more values are equivalent to the union of those values. So, Literal[-3, b"foo", MyEnum.A] is equivalent to Union[Literal[-3], Literal[b"foo"], Literal[MyEnum.A]]. This makes writing more complex types involving literals a little more convenient.

I think we can write `Literal['on', 'off', 'probe']` then?

Reply:
cool, thanks